libkbfs/block_journal.go

```diff
@@ -226,33 +229,40 @@ func aggregateInfoPath(dir string) string {
 	return filepath.Join(dir, "block_aggregate_info")
 }
 
-func (j *blockJournal) changeBytes(
-	deltaStoredBytes, deltaUnflushedBytes int64) error {
+func (j *blockJournal) changeCounts(
+	deltaStoredBytes, deltaStoredFiles, deltaUnflushedBytes int64) error {
 	j.aggregateInfo.StoredBytes += deltaStoredBytes
+	j.aggregateInfo.StoredFiles += deltaStoredFiles
```

Comment:
Yeah, true. We should bound it, anyway. Implemented saturated add and added tests for it.